packages/dropdowns/package.json

```diff
@@ -14,10 +14,10 @@
     "dist"
   ],
   "scripts": {
-    "build": "../../utils/scripts/build.sh",
-    "build:demo": "../../utils/scripts/build-demo.sh",
-    "start": "../../utils/scripts/start.sh"
```

Comment:
Removing `start` breaks `yarn start` (styleguidist) for dropdowns – not sure this is desired.